### ISSUE TITLE
`vagrant up` hangs at "Starting the VMware VM..." when using VMware Fusion

### ISSUE THREAD

**Issue by blomma** [mitchellh/vagrant#5379]
Vagrant Version: 1.7.2
Vagrant VMware Fusion plugin: 3.2.3

**Steps to reproduce:**

1. Restart mac
2. run vagrant up
3. Hangs on "Starting the VMware VM..." and never goes any further

**Workaround:**

Before running vagrant up for the first time start VMware Fusion. It only needs to be started once and then it works even if you close VMware Fusion again.

**Debug log**

https://gist.github.com/sethvargo/dae9bd7532c49cd92e29

**Comment by mitchellh** [comment]
Based on your log, it is simply waiting on `vmrun` to exit, which it hasn't. Can you run this and see if `vmrun` is actually still running? Also, stracing it would help to see what it is trying to do.

**Comment by jakerobinson** [comment]
@blomma did you by chance update fusion recently? It is possible that VM networking in fusion does not work after upgrading, and no VMs will boot. This in turn causes timeouts in Vagrant. To test your system, you can download a fresh CentOS (or your favorite distro) ISO, create a VM from it, and see if networking works.

To fix the networking portion, simply walk through the steps in this KB: http://kb.vmware.com/kb/2009642

**Comment by blomma** [comment]
@mitchellh Sorry for the late reply but I've had a hell of a time trying to reproduce this problem with trace running. This is what I've found so far.

On my MacBook Air this problem i described is 100% reproducible
On my Shiny new iMac Retina this problem is around 90% reproducible (i can't explain why)

With dtruss running i can never trigger this bug on either computer
This is the specific command I'm running

`sudo dtruss -f -t open sudo -u $USER vagrant up`

And lastly vmrun is still running when it is hanging.

`/Applications/VMware Fusion.app/Contents/Library/vmrun start /Users/blomma/Vagrant/unitas/.vagrant/machines/dev/vmware_fusion/6ffa78e8-3099-49b6-82a6-55ee0a4b08fc/windows8_x64_base.vmx gui`

@jakerobinson Interesting but alas not my problem, I've checked out the support doc and checked my network settings on both computers and according to the support doc they are both fine.
